### DISCUSSION TITLE
Unable to clean a corrupt session

### DISCUSSION THREAD

**Discussion by rajneesh2k10** [rmagatti/auto-session#368] [Q&A] [answered]
I ended up with a corrupted session in auto-session after some weird order of closing nvim buffer, neo-tree, and outline. Now, now matter what I do, when I start neovim in that directory I always get the following error (I've redacted some parts) - 

Note that auto-session is working fine in other directories.
```
51 buffers wiped out
Error detected while processing VimEnter Autocommands for "*":
auto-session ERROR: Error restoring session, disabling auto save.
Error: vim/_editor.lua:0: VimEnter Autocommands for "*"..script nvim_exec2() called at VimEnter Autocommands for "*":0../Users/***/.local/share/nvim/sessions/%2FUsers%2F***%2Fproject%2F
Fmanager.vim, line 81: Vim(fold):E16: Invalid range: 4,18fold
```

Here is what I have tried - 
1. Re-installed auto-sessions, neo-tree, outline, and nvim-ufo.
2. Removed the session from `/Users/xxx/.local/share/nvim/session`.
3. I have tried to re-enable auto-session in that directory (which is disabled as per the error message).

I still get the same error. 

I am unsure if the auto-session is saving the sessions at the location `/Users/xxx/.local/share/nvim/session` though because I cannot see ALL the sessions there. Also, I don't see the session corresponding to the one listed in the error message.

Any clues how can I debug it further or wipe all sessions and start fresh?

**Comment by cameronr** [comment, marked as answer]
By default, sessions are stored in `~/.local/share/nvim/sessions` (note the trailing `s`) so manually deleting the session file there will make the error go away.

You can also save a new session over the existing one via `:SessionSave` and as long as the error isn't in the current session, that will likely resolve it.

You can also delete the session for the current directory via `:SessionDelete`

Lastly, you can open the session picker via `:SessionSearch` and then delete the session by highlighting it and pressing ctrl-d.

**Reply by rajneesh2k10** [reply]
Wow. I have been looking at wrong directory all this time. I previously used `mini.sessions` that stored sessions in `~/.local/share/nvim/session`. I was cleaning up session from there instead - which of course didn't affect auto-sessions at all.

Thank you for your help!
